CMSgov/onemac
# OY2 22444: Verify ID format checkers use correct Regex for Waiver numbers.

Story: https://qmacbis.atlassian.net/browse/OY2-22444
Endpoint: See github-actions bot comment

### Details
In Regular Expressions, '.' means match any single character.  OneMAC ID formats should match specifically to '.' so the character needs to be escaped.

### Changes
- Updated all Regular Expressions using an unescaped (captured?) '.' are now escaped.

### Test Plan
1. Login as a submitting user
2. Navigate to Package Dashboard
3. Click New Submission->Waiver Action->1915(b) Waiver Renewal
4. Verify the ID format shown is enforced (specifically the '.' characters)
5. Click New Submission->Waiver Action->1915(b) Waiver Amendment
6. Verify the ID format shown is enforced (specifically the '.' characters)
7. Click New Submission->Waiver Action->1915(c) Appendix K Amendment
8. Verify the ID format shown is enforced (specifically the '.' characters)
9. Force an Appendix K package into RAI Issued status
10. Click the "Respond to RAI" action on that package
11. Verify the ID format shown is enforced (specifically the '.' characters)
